xray/test/test_dataarray.py

```diff
@@ -67,6 +67,18 @@ def test_name(self):
         expected = DataArray(Coordinate('y', [3]))
         self.assertDataArrayIdentical(actual, expected)
 
+        te = (TypeError, 'string or NoneType')
+        ve = (ValueError, 'string must be length 1 or')
+        for name, e in zip([0, (4, ), True], [te, te, te, ve]):
```

Comment:
nice catch.  fixed. I wish zip complained when the length of the arguments are not the same.